src/css/index.scss

```diff
@@ -0,0 +1,230 @@
+* {
```

Comment:
Good job! 👍 You may wanna have a look more about the scss syntax, and we will teach this latter